packages/create-remix/templates/express/server.js

```diff
@@ -1,21 +1,35 @@
+import path from "path";
 import express from "express";
 import compression from "compression";
 import morgan from "morgan";
 import { createRequestHandler } from "@remix-run/express";
 
 import * as build from "@remix-run/dev/server-build";
 
+const PUBLIC_DIR = path.join(process.cwd(), "public");
+const BROWSER_BUILD_DIR = "/build/";
```

Comment:
won't this break the ability to configure [`publicPath`](https://remix.run/docs/en/v1/api/conventions#publicpath)? 🤔